### ISSUE TITLE
Is this project Abandoned?

### ISSUE THREAD

**Issue by violentmagician** [GetBlimp/django-rest-framework-jwt#449]
Hi,
The Last commit on this project is from Oct 2017.
Based on the number of issues and open Pull requests this project looks abandoned to me/

Can a maintainer confirm the case?

**Comment by dillonreyna** [comment]
I've just emailed Blimp. Will update if they respond.

**Comment by ksitko** [comment]
Any work on the status of the project?

**Comment by erayaydin** [comment]
Are there any status change? If not, we should move project. What is latest status?

**Comment by violentmagician** [comment]
I don't have a response from them.

**Comment by wakaryry** [comment]
could try [django-rest-framework-simplejwt](https://github.com/davesque/django-rest-framework-simplejwt).

**Comment by BugDeveloper** [comment]
![Meh](https://i.imgflip.com/2g1gaq.jpg)

**Comment by dillonreyna** [comment]
Yep - still no response to my email after 5 weeks. I think we can call this one dead.

**Comment by WisdomPill** [comment]
@GetBlimp, @jpadilla or @jlhernandez1388 can you please let us know if the project has any future? I noticed that in the django rest framework website they suggest another library now for jwt, maybe this has to do with lack of maintenance in the last year

**Comment by kylecribbs** [comment]
This project had a lot of potential!

**Comment by Alex3917** [comment]
@kylecribbs It works fine. If it stops working in the future I'm sure someone will just fork it. No one is going to spend a month rewriting their auth system rather than just spending five minutes to fix any issues that crop up with Python 3.8 or whatever.

**Comment by kylecribbs** [comment]
@Alex3917 it doesn’t support Django 2.0. Django 3 is coming out next year with 1.11 becoming end of life.. So it’s pretty safe to call this dead.

**Comment by jayvdb** [comment]
https://github.com/Styria-Digital/django-rest-framework-jwt is a drop-in updated replacement fork

**Comment by jheld** [comment]
The maintainer is willing to transfer ownership (including new pypi project under same name, etc). Please get in touch with him or check the newest status issue to help.

**Comment by jpadilla** [comment]
Thank you all for this conversation. I've just posted an update https://github.com/jpadilla/django-rest-framework-jwt/issues/484
